app/permissions.js

```diff
@@ -0,0 +1,21 @@
+const permissions = {
+  viewPaymentHolds: 'Payments.View.Holds',
+  viewPaymentScheme: 'Payments.View.Scheme',
+  updatePaymentScheme: 'Payments.Update.Scheme',
+  addPaymentHold: 'Payments.Add.Hold',
+  removePaymentHold: 'Payments.Remove.Hold'
```

Comment:
Not really in the scope of this PR but we will need to refine these roles with the business.

Reply:
Added these role for testing only